src/libraries/Common/tests/TestUtilities/System/WindowsIdentityFixture.cs

```diff
@@ -118,12 +121,15 @@ private void CreateUser()
         [return: MarshalAs(UnmanagedType.Bool)]
         private static partial bool LogonUser(string userName, string domain, string password, int logonType, int logonProvider, out SafeAccessTokenHandle safeAccessTokenHandle);
 
-        [DllImport("netapi32.dll", SetLastError = true)]
-        internal static extern uint NetUserAdd([MarshalAs(UnmanagedType.LPWStr)] string servername, uint level, ref USER_INFO_1 buf, out uint parm_err);
+        [LibraryImport("netapi32.dll", SetLastError = true)]
+        internal static partial uint NetUserAdd([MarshalAs(UnmanagedType.LPWStr)] string servername, uint level, ref USER_INFO_1 buf, out uint parm_err);
 
         [LibraryImport("netapi32.dll")]
         internal static partial uint NetUserDel([MarshalAs(UnmanagedType.LPWStr)] string servername, [MarshalAs(UnmanagedType.LPWStr)] string username);
 
+#if NET7_0_OR_GREATER
+        [NativeMarshalling(typeof(USER_INFO_1Marshaller))]
```

Comment:
```suggestion
        [NativeMarshalling(typeof(USER_INFO_1.Marshaller))]
```

@pedrobsaila Could this avoid `CS0104` with the name `Marshaller`?